scripts/build-dev-html.sh

```diff
@@ -1,46 +1,51 @@
-#!/bin/sh
+#!/bin/bash
 NOT='.sh$|^.git|.jsx$|^tests|^.eslint|^.travis|^package.json$|^package-lock.json$'
 DIST=dist-dev-html
 VENDOR_DIR=$DIST/content/vendor
 ADDON_DIR=addon
 DEV_SERVER_PORT=8126
 
+WATCH_FLAG=""
+if [ $1 = "--watch" ]
```

Comment:
On bash on Mac, I'm seeing `scripts/build-dev-html.sh: line 9: [: =: unary operator expected`

Reply:
Ah, apparently, it should be `"$1"` to work on older BASH versions.